apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.IntegrationTests/Trace/CloudTraceTest.cs

```diff
@@ -98,14 +99,106 @@ public void Trace_Multiple_Spans()
             Assert.Equal(2, trace.Spans.Count);
         }
 
-        [Fact(Skip = "Until we have CI infrastructure to deploy Google.Cloud.Diagnostics.AspNet.ClientApp")]
-        public async Task Trace_CurrentTracer_AsyncLocal()
+        // All the following tests would need an AspNet application to be deployed to work,
+        // and we currently don't have such a setup.
+        // The application we are using is Google.Cloud.Diagnostics.AspNet.ClientApp and we deploy
+        // on local and test on local.
+        // But for CI all these tests should be skipped. See this comment as to why:
+        // https://github.com/googleapis/google-cloud-dotnet/issues/2167#issuecomment-409605030
+
+        private static readonly Uri PublishedServiceBaseAddress = new Uri("http://localhost:58755");
+
+        [Fact(Skip = "https://github.com/googleapis/google-cloud-dotnet/issues/2167#issuecomment-409605030")]
+        public async Task Trace_Iss()
+        {
+            var service = $"/api/Trace/Trace/{_testId}";
+            var childSpanName = EntryData.GetMessage("Trace", _testId);
+
+            HttpResponseMessage response;
+            using (HttpClient httpClient = new HttpClient())
+            {
+                httpClient.BaseAddress = PublishedServiceBaseAddress;
+                response = await httpClient.GetAsync(service);
+            }
+
+            var trace = _polling.GetTrace(service, _startTime);
+
+            TraceEntryVerifiers.AssertParentChildSpan(trace, service, childSpanName);
+            TraceEntryVerifiers.AssertSpanLabelsContains(
+                trace.Spans.First(s => s.Name == service), TraceEntryData.HttpGetSuccessLabels);
+
+            Assert.False(response.Headers.Contains(TraceHeaderContext.TraceHeader));
+        }
+
+        [Fact(Skip = "https://github.com/googleapis/google-cloud-dotnet/issues/2167#issuecomment-409605030")]
+        public async Task Trace_Iss_Label()
+        {
+            var service = $"/api/Trace/TraceLabels/{_testId}";
+            var childSpanName = EntryData.GetMessage("TraceLabels", _testId);
+
+            using (HttpClient httpClient = new HttpClient())
+            {
+                httpClient.BaseAddress = PublishedServiceBaseAddress;
+                await httpClient.GetAsync(service);
+            }
+
+            var trace = _polling.GetTrace(service, _startTime);
+
+            TraceEntryVerifiers.AssertParentChildSpan(trace, service, childSpanName);
+            TraceEntryVerifiers.AssertSpanLabelsContains(trace.Spans.First(s => s.Name == childSpanName),
+                new Dictionary<string, string>
+                {
+                        {TraceEntryData.TraceLabel, TraceEntryData.TraceLabelValue }
```

Comment:
Nit: odd indentation, and add a space after {

Reply:
Done